config/projects.yml

```diff
@@ -214,11 +214,10 @@ doctrine-mongodb-admin-bundle:
 doctrine-orm-admin-bundle:
   branches:
     master:
-      php: ['7.2', '7.3', '7.4']
+      php: ['7.3', '7.4']
       target_php: '7.3'
       versions:
         symfony: ['4.4']
-        sonata_admin: ['3']
```

Comment:
Should we add `dev-master` to test against the master branch?

Reply:
The composer.json is already using dev-master, this would be a duplicate build then.